### DISCUSSION TITLE
How to hide the User Directory from a custom role user?

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#11212] [Archive]
I have created a new role called `Custom Role` with `App Access` enabled.
As you can see in the screenshot, the users with that role, can see the User Directory:
<img width="1241" alt="Screenshot 2022-01-23 at 02 08 01" src="https://user-images.githubusercontent.com/1007051/150660682-4cb32044-8565-4c5f-9be2-f25a64715754.png">

How can I hide it? I just want them to be able to see the content.

I would also like to be able to hide the link to the documentation. Again, hide only for this role.

Thanks!

**Comment by joselcvarela** [comment]
In order to restrict role to see other users you can one of two things:
- `Reset System Permissions to: App Access Minimum`
![image](https://user-images.githubusercontent.com/14039341/150690448-7aab5d21-bbef-4071-b40f-78a043c999d4.png)

- Set permissions to `None` in `Directus Users`.


About documentation, in the left side bar it's not possible to restrict items per role. Instead you can show/hide/organise them globally:
![image](https://user-images.githubusercontent.com/14039341/150690437-74d78ed1-553c-47be-8a99-e3ff2d706d7c.png)

**Reply by abdonrd** [reply]
- I already have the minimum `App Access` permissions.

- I can't set `No access` to the `Directus Users`:
<img width="833" alt="Screenshot 2022-01-23 at 19 17 25" src="https://user-images.githubusercontent.com/1007051/150692208-90613f65-bd8a-4085-870c-295032439934.png">

**Reply by joselcvarela** [reply]
If you set the permissions like this then the user will only be able to see himself on user directory.
To hide User Directory, you will need to do the same like you did for the Documentation. But, it is also globally.

**Reply by abdonrd** [reply]
Yeah, but what I'm looking for is that...

The User Directory appears for people who have permissions (like Admins, etc):
<img width="1278" alt="Screenshot 2022-01-24 at 13 01 55" src="https://user-images.githubusercontent.com/1007051/150779560-03f68c3c-91b1-4813-a35d-618baf9ae2e4.png">

And not appear for users who can only edit their profile (like this custom role with just minimum `App Access` permissions):
<img width="1278" alt="Screenshot 2022-01-24 at 12 51 11" src="https://user-images.githubusercontent.com/1007051/150779597-479932cf-762d-4a0b-90f4-a5050fe2b1cd.png">

  As you can see, there is no references to a Users Directory anywhere.

**Reply by joselcvarela** [reply]
Yeah I know, this is currently not possible. As I mentioned on the other discussion, there's already a feature request to customise the Module Bar per role

https://github.com/directus/directus/discussions/11217#discussioncomment-2031284

**Comment by abdonrd** [comment]
As @joselcvarela mention here https://github.com/directus/directus/discussions/11212#discussioncomment-2034420

This is currently **not** possible.
Today latest Directus version is `v9.6.0`.

I have created a feature request here: https://github.com/directus/directus/discussions/11923
